engine/src/main/java/org/terasology/config/flexible/FlexibleConfigImpl.java

```diff
@@ -36,16 +36,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(FlexibleConfigImpl.class);
     private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
 
-    private Map<SimpleUri, Setting> settings;
-    private Map<SimpleUri, String> temporarilyParkedSettings;
-
-    /**
-     * Creates a new {@link FlexibleConfigImpl} instance.
-     */
-    public FlexibleConfigImpl() {
-        this.settings = Maps.newHashMap();
-        this.temporarilyParkedSettings = Maps.newHashMap();
-    }
+    private Map<SimpleUri, Setting> settings = Maps.newHashMap();
+    private Map<SimpleUri, String> temporarilyParkedSettings = Maps.newHashMap();
```

Comment:
Morale behind shifting this here: When something is initialized like this, it's clear at the first glance that it can never be null. However, if it's initialized in the constructor, that raises the question of whether this can ever be null, and whether checks for the same should be added.

(P.S. I could not find any real reason to prefer one style over the other.)

Reply:
...hmmm... the code could still assign null (intentionally or by mistake) to such variable. 

It's no big deal though, let's not worry about this anymore.

Reply:
True. I was sorely tempted to make it final as well in the same line, but I refrained.

Not a big deal, but can't deny that I am tempted :D

Reply:
Admittedly, why not final?